.test-infra/terraform/google-cloud-platform/google-kubernetes-engine/beam-utility.apache-beam-testing.tfvars

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+project    = "apache-beam-testing"
+network    = "default"
+subnetwork = "default-f91f013bcf8bd369"
+region     = "us-central1"
+router     = "default-us-central1-router"
+router_nat = "default-us-central1-router-nat"
+cluster_name_override = "beam-utility"
```

Comment:
Could we name this something more specific?

Reply:
Think we should keep it this as we should add more to this cluster instead of creating multiple

Reply:
Because auto pilot scales to the workload, we can have multiple clusters focused on a specific resource need. That's the reason for having this re-usable GKE auto pilot creating solution. I'd argue that `beam-utility` will not make sense to someone trying to fix or add to the infrastructure later.

Reply:
Would kafka-workflows be precise enough?